src/bramble/client/StateManager.js

```diff
@@ -106,9 +106,14 @@ define(function() {
                 get: function()  { return getBool(storage, "allowJavaScript"); },
                 set: function(v) { storage.setItem(prefix("allowJavaScript"), v); }
             },
+            editHint: {
+                get: function()  { return getBool(storage, "editHint"); },
+                set: function(v) { storage.setItem(prefix("editHint"), v); }
+
```

Comment:
this looks like it's missing a `},` to separate it from autoUpdate, which means it'll fail when you try a `run build`